src/handle_provide_parameter.c

```diff
@@ -100,7 +100,6 @@ static uint8_t prepare_reading_next_input(context_t *context) {
                 PRINTF("Preparing to read V3_SWAP_EXACT_OUT\n");
                 context->next_param = INPUT_V3_SWAP_EXACT_OUT_LENGTH;
                 break;
-            case PERMIT2_TRANSFER_FROM:
```

Comment:
so it seems that we can remove the enum in this file and all the things related in test (uniswap_crafter.py)